### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by blessing-michael**
In this project I;
- Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Added CSS styling to bookstore app

Please note:
These request was optional but however I;
-Added --black-color: #121212; and reused code where necessary

### CODE REVIEW

File: src/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+  gap: 3rem;
+}
+
+.book-category {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: var(--Montserrat);
+  font-size: 1.2rem;
+  font-weight: bold;
+  color: #121212;
```
Comment (1GORDON) [inline src/styles/styles.css]: Great job @blessing-michael 🤝

- [ ] Kindly ensure you do not repeate the same rule. You may create a single class for the repeating rule and re-use this class where necessary. You may do this for the `color: #121212;`rule, and for the rules I could have missed out to see.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @blessing-michael ✋

Good job so far!

## Highlights!
- No usage of the ID selector ✔️
- No use of Important keyword✔️
- Your design matches the design provided ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly ensure the MIT license provided is for this exact project.
MiT-license


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/styles.css (hunk 1 above)
Comment (thecodechaser) [inline src/styles/styles.css]: - [ ] As requested by the previous reviewer, Please make sure to not repeat the same rule in CSS. You should create a single class for the repeating rule and re-use this class in react code where necessary. You Should do this for the `color: #121212;` rule.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @blessing-michael👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Nice UI design. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/styles.css (hunk 1 above)
Comment (blessing-michael) [reply src/styles/styles.css]: @thecodechaser Thank you for the review. I would like to mention that, the requested changes is optional and not mandatory according to the projects requirement. Kindly look into it. Thank you

**Review by frank1738 — APPROVED** [review]
Hi @blessing-michael , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Good work working on the optional change to make your code clean
✅ App is designed as expected 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**